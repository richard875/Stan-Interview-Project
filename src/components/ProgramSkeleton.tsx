import styled from "styled-components";

const ProgramSkeleton = () => (
  <Container>
    <Card />
    <Right>
      <Line1 />
      <Line2 />
      <Line3 />
    </Right>
  </Container>
);

export default ProgramSkeleton;

const Container = styled.div`
  gap: 50px;
  height: 315px;
  display: flex;
  align-items: center;
  margin-top: 50px;
`;

const Card = styled.div`
  width: 200px;
  height: 300px;
  background-color: #5b5555;
`;

const Right = styled.div`
  gap: 10px;
  height: 100%;
  display: flex;
  flex-direction: column;
`;

const Line1 = styled(Card)`
  width: 150px;
  height: 30px;
`;

const Line2 = styled(Card)`
  width: 400px;
  height: 25px;
`;

const Line3 = styled(Card)`
  width: 630px;
  height: 130px;
`;
