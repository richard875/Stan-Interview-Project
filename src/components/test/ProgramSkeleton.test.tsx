import renderer from "react-test-renderer";
import ProgramSkeleton from "../ProgramSkeleton";

it("renders correctly", () => {
  const component = renderer.create(<ProgramSkeleton />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
