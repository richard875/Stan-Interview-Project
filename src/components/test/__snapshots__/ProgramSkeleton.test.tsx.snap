// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-beqWNU gJfEHF"
>
  <div
    className="sc-gueXAH XXvTF"
  />
  <div
    className="sc-dmqGRC ieelUQ"
  >
    <div
      className="sc-gueXAH sc-hLseSp XXvTF gSYLRh"
    />
    <div
      className="sc-gueXAH sc-eDDOjk XXvTF hpNEpH"
    />
    <div
      className="sc-gueXAH sc-jTrOVX XXvTF kRSHip"
    />
  </div>
</div>
`;
